boards/shared/Src/App/App_SharedHeartbeatMonitor.c

```diff
@@ -42,7 +42,7 @@ bool App_SharedHeartbeatMonitor_Tick(struct HeartbeatMonitor *const heartbeat_mo
 
     if ((current_ms - heartbeat_monitor->previous_timeout_ms) >= heartbeat_monitor->timeout_period_ms)
     {
-        heartbeat_monitor->previous_timeout_ms += heartbeat_monitor->timeout_period_ms;
+        heartbeat_monitor->previous_timeout_ms = current_ms;
 
```

Comment:
The previous line was making it hard to run tests, and I think the new line makes more sense anyway? Could you a double check here

Reply:
can't see why the functionality would be affected with this change, so if it helps with testing looks fine to me